src/dev-app/theme.scss

```diff
@@ -41,6 +42,7 @@ $candy-app-theme: mat-light-theme((
 @include angular-material-mdc-theme($candy-app-theme);
 @include mat-column-resize-theme($candy-app-theme);
 @include mat-popover-edit-theme($candy-app-theme);
+@include mat-mdc-slider-theme($candy-app-theme);
```

Comment:
Can you add a super-small comment on why we do this separately to the MDC theme?